src/handlers/instance.rs

```diff
@@ -65,6 +66,143 @@ pub async fn get_instance_info(
     Ok(Json(instance.get_instance_info().await))
 }
 
+pub async fn create_minecraft_bedrock_instance(
```

Comment:
you should add a variant to the `Game` enum, and dispatch accordingly in `create_minecraft_instance`